Add info about vss distribution (#366)
cfrg/draft-irtf-cfrg-frost: draft-irtf-cfrg-frost.md

It is assumed the dealer then sends one secret key share to each of the `NUM_PARTICIPANTS` participants, along with `vss_commitment`.
After receiving their secret key share and `vss_commitment`, participants MUST abort if they do not have the same view of `vss_commitment`.

> Ensuring a consistent view of this VSS commitment can be achieved by employing a secure broadcast channel.

Reviewer: ```suggestion
The dealer can use a secure broadcast channel to ensure each participant has a consistent view of this commitment.
```

Reviewer 2: I don't get this - the dealer already sends the private key and commitments though a secure channel. What else is needed here? I kinda assumed that participants had to talk between themselves to check if they have the same commitments, but I'm not sure.

Author: Exactly, but if they talk amongst themselves to check if they have the same commitments, it has be done in a way that it fulfills the properties of a broadcast channel. Otherwise, some players could cheat and send inconsistent commitments, etc.
